src/core/wrap.ts

```diff
@@ -11,5 +13,5 @@ export type IfSingle<T extends string, Yes, No> = StripEscapes<T> extends `${inf
 
 export const wrap = (s: string | Input<any>) => {
```

Comment:
```suggestion
const NEEDS_WRAP_RE = /^(\(.*\)|\\?.)$/

export const wrap = (s: string | Input<any>) => {
```

Reply:
got it, thanks for pointing this out!  📓✍️